src/main/java/com/example/bookstoreapp/services/impl/CatalogItemServiceImpl.java

```diff
@@ -1,27 +1,37 @@
 package com.example.bookstoreapp.services.impl;
 
+import com.example.bookstoreapp.entities.CatalogItemEntity;
 import com.example.bookstoreapp.models.CatalogItem;
+import com.example.bookstoreapp.repositories.CatalogItemEntityRepository;
 import com.example.bookstoreapp.services.CatalogItemService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
 import java.util.List;
 
 
 @Service
 public class CatalogItemServiceImpl implements CatalogItemService {
 
+
+    @Autowired
+    private CatalogItemEntityRepository catalogItemEntityRepository;
     @Override
     public CatalogItem create(CatalogItem catalogItem) {
         catalogItem.setId(System.currentTimeMillis());
```

Comment:
Id is currentTimeMillis

Reply:
catalogItem.setId(System.currentTimeMillis());  It means id will save within millisecond . is not it

Reply:
No. 

We are assigning the id = System.currentTimeMillis()